### ISSUE TITLE
PF4: Progress TypeError: has no exported member 'ProgressSize'

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#782]
Attempted to pull the latest from PF4 react-core (v1.23.0), pf-next (1.0.57 and 1.0.58) and encountered the following build error. This prevents koku-ui from rendering.

![screen shot 2018-10-16 at 11 31 19 am](https://user-images.githubusercontent.com/17481322/47028267-0b6e8900-d137-11e8-8067-42e1099e86ee.png)

In addition, it appears that only 'xs', 'md', and 'lg' are known to the component.

Usage:
```
import { Progress, ProgressSize } from '@patternfly/react-core';
...
<Progress value={Number(percent.toFixed(2))} title="Descriptive text here" size={ProgressSize.sm} />
```

**Comment by karelhala** [comment]
Thanks for reporting this, it looks like I made a mistake when exporting these props in typescript, in meantime you can use it without ProgressSize
```TSX
import { Progress } from '@patternfly/react-core';
...
<Progress value={Number(percent.toFixed(2))} title="Descriptive text here" size="sm" />
```

**Comment by dlabrecq** [comment]
My IDE tells me that 'sm' is not valid -- only 'xs', 'md', and 'lg' are available.

**Comment by dlabrecq** [comment]
I took the latest react-core v1.24.0 package and get this error now.

![screen shot 2018-10-16 at 2 57 23 pm](https://user-images.githubusercontent.com/17481322/47040271-e0def900-d153-11e8-8f2c-e9c6917b763e.png)

That said, this latest react-core package is causing a lot more errors. I had to drop back to v1.23.1 just to compile.

![screen shot 2018-10-16 at 2 59 51 pm](https://user-images.githubusercontent.com/17481322/47040531-71b5d480-d154-11e8-89ef-ede49687be49.png)
